tests/unit/aiplatform/test_helpers.py

```diff
@@ -147,3 +147,45 @@ def test_invalid_prediction_uri_args(self, args, expected_error_msg):
             )
 
         assert err.match(expected_error_msg)
+
```

Comment:
(Comment not particular to just this file)

There are a number of code test coverage gaps in files introduced by this PR.
We're currently in an SDK-wide initiative to get code coverage back over 99% overall.

Here are some files introduced in this PR with more than 1 line of missing test coverage. You don't necessarily need to hit every single one of these, but it would be helpful if you introduced some small, targeted unit tests to hit some of the larger gaps.

google/cloud/aiplatform/docker_utils/build.py                                                                          86      3     32      2    96%   182, 522-530
google/cloud/aiplatform/docker_utils/run.py                                                                            69      2     28      0    98%   26-27
google/cloud/aiplatform/docker_utils/utils.py                                                                          17      2      0      0    88%   22-23
google/cloud/aiplatform/prediction/handler.py                                                                          31      4      6      0    89%   24-25, 53, 66
google/cloud/aiplatform/prediction/handler_utils.py                                                                    33      3     20      1    92%   22-23, 44
google/cloud/aiplatform/prediction/local_endpoint.py                                                                  152     32     46      1    78%   194-196, 277-281, 294-303, 311-341, 393->398
google/cloud/aiplatform/prediction/predictor.py                                                                        15      3      2      0    82%   34, 44, 69
google/cloud/aiplatform/prediction/serializer.py                                                                       34      4      8      0    90%   23-24, 49, 64
google/cloud/aiplatform/utils/prediction_utils.py                                                                      81      7     24      1    90%   382-392

Reply:
You can run "nox -s cover" to see test coverage